mm-common/PKGBUILD

```diff
@@ -43,5 +37,3 @@ package() {
   cd "${srcdir}/build-${CARCH}"
   DESTDIR="${pkgdir}" meson install
 }
-
-# vim:set ts=2 sw=2 et:
```

Comment:
😆 is there some style guide about vim/emacs modeline comments.  I added one in the split package template, so they are multiplying.

Reply:
AFAIK, there is no style guidelines in any msys2 repo.